### PR TITLE
add in jump to date button to calendars where it is relevant

### PR DESCRIPTION
**Pull request by Guntrisoft**
screen shot 2016-11-01 at 15 46 59

### CODE REVIEW

File: app/assets/javascripts/calendar.es6 [hunk 1]
```diff
@@ -11,6 +11,21 @@ class Calendar {
     const defaultConfig = {
       allDaySlot: false,
       cookieName: this.$el.attr('id') || 'calendar',
+      customButtons: {
+        jumpToDate: {
+          text: 'Jump to date',
+          click: (event) => {
+            this.$jumpToDateEl.val(this.getCurrentDate());
+
+            const dateRangePicker = this.$jumpToDateEl.data('daterangepicker');
```
Comment (benbarnett) [inline app/assets/javascripts/calendar.es6]: Can we try to keep the variable declarations at the top of each method?

File: app/assets/javascripts/calendar.es6 [hunk 2]
```diff
@@ -31,6 +46,34 @@ class Calendar {
     );
 
     this.$el.fullCalendar(this.config);
+
+    this.insertJumpToDate();
+  }
+
+  insertJumpToDate() {
+    const jumpToDateButton = $('.fc-jumpToDate-button');
```
Comment (benbarnett) [inline app/assets/javascripts/calendar.es6]: Should we prefix with `$` as it's a jQuery object?

**Comment by AndrewVos** [comment]
Looks good. Wondering if tests are worth the effort?

File: app/assets/javascripts/calendar.es6 [hunk 3]
```diff
@@ -11,6 +11,21 @@ class Calendar {
     const defaultConfig = {
       allDaySlot: false,
       cookieName: this.$el.attr('id') || 'calendar',
+      customButtons: {
+        jumpToDate: {
+          text: 'Jump to date',
+          click: (event) => {
```
Comment (benbarnett) [inline app/assets/javascripts/calendar.es6]: I know we don't have tests yet but if this was moved into it's own named method it'll make it easier to test & debug when we do.

File: app/assets/javascripts/calendar.es6 [hunk 4]
```diff
@@ -11,6 +11,21 @@ class Calendar {
...
+            this.$jumpToDateEl.val(this.getCurrentDate());
+
+            const dateRangePicker = this.$jumpToDateEl.data('daterangepicker');
+
+            dateRangePicker.setStartDate(this.getCurrentDate());
+            dateRangePicker.setEndDate(this.getCurrentDate());
```
Comment (benbarnett) [inline app/assets/javascripts/calendar.es6]: Worth declaring `this.getCurrentDate()` as a variable as we're referencing it 3 times?

File: app/assets/javascripts/calendar.es6 (hunk 3 above)
Comment (Guntrisoft) [inline app/assets/javascripts/calendar.es6]: done

File: app/assets/javascripts/calendar.es6 (hunk 1 above)
Comment (Guntrisoft) [inline app/assets/javascripts/calendar.es6]: done

**Pull request merged**